src/android/Print.java

```diff
@@ -725,13 +725,20 @@ private int getPrinterSeries(String target) {
 	private void sendToPrintConnect(JSONArray args, final CallbackContext callbackContext) throws JSONException {
 		final String zplString = args.getString(0);
 		byte[] passthroughBytes = null;
+		boolean requiresForeground = false;
 
 		try {
 			passthroughBytes = zplString.getBytes("UTF-8");
 		} catch (UnsupportedEncodingException e) {
 			// Handle exception
 		}
 
+		PackageInfo info = this.mContext.getPackageManager().getPackageInfo("com.zebra.printconnect", 0);
+		long versionCode = info.getLongVersionCode();
+		if (versionCode > 0) { // TODO
```

Comment:
What is this a TODO for?

Reply:
Testing purposes -- this 0 should be the actual version threshold, but I don't know what that should be yet.